IssueCreator/IssueCreator/Settings.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.Json;
+
+namespace IssueCreator
+{
+    public class Settings
+    {
+        private static readonly byte[] s_entropy = { 23, 61, 24, 8, 77, 52 }; //the entropy
```

Comment:
Thank you for my new lucky Powerball numbers!

Reply:
Split the win!